sectionfilter.c

```diff
@@ -270,54 +271,68 @@ cSatipSectionFilterHandler::~cSatipSectionFilterHandler()
       Delete(i);
 }
 
+void cSatipSectionFilterHandler::SendAll(void)
+{
+  while (true) {
```

Comment:
The infinite loop is a bad design habit. I think "do {} while (available)" would be pretty much perfect here or even "for (i=0; i<100;++i) break;" where number of maximum loops is a finite.

Reply:
What about using
<pre>
while (Running()) {}
</pre>
it would also handle the case of infinite looping.

Reply:
Hmm,... Thought a second time about it. I don't even see the case of infinite looping.